js/main.js

```diff
@@ -295,6 +295,7 @@ var MM = (function () {
 		// Otherwise cancel show action.
 		if (module.lockStrings.length !== 0 && options.force !== true) {
 			Log.log("Will not show " + module.name + ". LockStrings active: " + module.lockStrings.join(","));
+			callback("Active lock strings");
```

Comment:
Maybe rename it to "ERR_ACTIVE_LOCK_STRINGS" so it is a little clearer that it is a error string?

Reply:
Er even better: allow for an optional error callback?

Reply:
I changed the error for the callback for an actual error object. If you log the error it gives you an actual stack trace from its origin which helps to debug.

![image](https://user-images.githubusercontent.com/9334168/106194076-a972aa00-61ae-11eb-9477-b26c2ff2ac28.png)

I think adding a second callback just for the error isn't necessary. Also, @rejas just fixed the callback recently. Until now it was never triggered, so we are not breaking existing modules with it.

Reply:
The problem of this is older modules might not check if the callback contains an error. They might think the action was executed successfully because the callback was called. So this is a somewhat breaking change.

Reply:
@MichMich if you want me to change it I can, but as I said, @rejas just fixed the callback which means until now the callback was never executed at all https://github.com/MichMich/MagicMirror/pull/2385/files#diff-50260f11109f55b944da641c51de244dffd718cbd77f7e5ce7b9124ab18fe265R437

Reply:
Ah, true. But still I think a separate error callback is a bit cleaner (and easier to uses as a module developer). Or do you disagree?